src/transformers/models/time_series_transformer/configuration_time_series_transformer.py

```diff
@@ -112,12 +112,12 @@ class TimeSeriesTransformerConfig(PretrainedConfig):
         Example:
 
     ```python
-    >>> from transformers import TimeSeriesTransformerConfig, TimeSeriesTransformerModel
+    >>> from transformers import TimeSeriesTransformerModel, TimeSeriesTransformerConfig
```

Comment:
Config should go before Model, which is already the case in the current `main` branch. No need to change here.

Reply:
okay.

Reply:
My comment here  is not addressed yet.

Reply:
Sorry, previously I missed it. now it is done